src/uct/sm/mm/mm_ep.c

```diff
@@ -14,39 +14,79 @@ SGLIB_DEFINE_HASHED_CONTAINER_FUNCTIONS(uct_mm_remote_seg_t,
                                         UCT_MM_BASE_ADDRESS_HASH_SIZE,
                                         uct_mm_remote_seg_hash)
 
+void uct_mm_ep_signal_remote_slow_path_callback(ucs_callbackq_slow_elem_t *self)
+{
+    uct_mm_ep_t *ep = ucs_container_of(self, uct_mm_ep_t, cbq_elem);
+
+    uct_mm_ep_signal_remote(ep, UCT_MM_IFACE_SIGNAL_CONNECT);
+}
 
 /* send a signal to remote interface using Unix-domain socket */
-static ucs_status_t
+ucs_status_t
 uct_mm_ep_signal_remote(uct_mm_ep_t *ep, uct_mm_iface_conn_signal_t sig)
 {
     uct_mm_iface_t *iface = ucs_derived_of(ep->super.super.iface, uct_mm_iface_t);
     int ret;
 
     /**
-     * Send connect message to remote interface
+     * Send connect/disconnect message to remote interface
```

Comment:
you mean to remove appearances of UCT_MM_IFACE_SIGNAL_DISCONNECT right? not CONNECT